AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/DataStoreConfiguration.swift

```diff
@@ -19,9 +19,9 @@ public struct DataStoreConflictData {
     public let remote: Model
 }
 
-/// Conflict Handler function typealias. The function is used during a conflict that
-/// could not be resolved and requires a decision from the consumer.
-public typealias DataStoreConflictHandler = (DataStoreConflictData, DataStoreConflictHandlerResolver) -> Void
+/// could not be resolved and requires a decision from the consumer. The consumer can call other methods (optionally
+/// asynchronous) before calling the escaping `DataStoreConflictHandlerResolver` with the resolution,
```

Comment:
How about something like..

> The DataStoreConflictHandler is an asynchronous callback which allows consumers to decide how to resolve conflicts between the frontend and backend.  This can be configured by....

Reply:
thanks, improving this